shell/src/main/java/alluxio/cli/fsadmin/journal/QuorumElectCommand.java

```diff
@@ -118,7 +227,44 @@ public String getDescription() {
 
   @Override
   public Options getOptions() {
-    return new Options().addOption(ADDRESS_OPTION_NAME, true,
-            "Server address that will take over as leader");
+    return new Options()
+        .addOption(ADDRESS_OPTION_NAME, true,
+            "Server address that will take over as leader")
+        .addOption(DOMAIN_OPTION_NAME, true, "Journal domain");
+  }
+
+  private RaftClient createClient() {
+    RaftProperties properties = new RaftProperties();
+    Parameters parameters = new Parameters();
+    RaftClientConfigKeys.Rpc.setRequestTimeout(properties,
+        TimeDuration.valueOf(15, TimeUnit.SECONDS));
+    ExponentialBackoffRetry retryPolicy = ExponentialBackoffRetry.newBuilder()
+        .setBaseSleepTime(TimeDuration.valueOf(100, TimeUnit.MILLISECONDS))
+        .setMaxAttempts(10)
+        .setMaxSleepTime(
+            TimeDuration.valueOf(100000, TimeUnit.MILLISECONDS))
```

Comment:
Are these default values good enough or there's a chance a user may tune them?

Reply:
Would you like to keep hardcode here until someone want to change it?